test/recipes/controls/aws_parallelcluster_install/openssh_spec.rb

```diff
@@ -25,10 +23,15 @@
   end
 
   files = %w(/etc/ssh/sshd_config /etc/ssh/ca_keys /etc/ssh/revoked_keys)
+  file_permission = if os.debian?
+                      '0644'
+                    else
+                      '0600'
+                    end
```

Comment:
Just out of curiosity: why?

Reply:
I verified it in the OS and this is the default. 
Not sure why in Ubuntu this is not aligned with RHEL but the root group has the possibility to read these files.